Airports/EGSU/Basic.txt

```diff
@@ -0,0 +1,3 @@
+Duxford
+N052.05.27.015 E000.07.56.506
```

Comment:
Assum this is taken from a .sline grab, can we use the ARP co-ordinates found in the AIP textual data?